deepnote_toolkit/sql/sql_execution.py

```
@@ -626,6 +626,22 @@ def _cancel_cursor(cursor: "DBAPICursor") -> None:
         pass  # Best effort, ignore all errors
 
 
+def _extract_rls_user_email(query):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_extract_rls_user_email`

(ANN202)